techniques/systemSettings/misc/setVars/1.0/setVars.st

```diff
@@ -0,0 +1,100 @@
+# Define one or more variable on rudder
+#   -Variables can be defined by a Json  file or a text input
+#    -Allowed types are string and dict
+
+bundle agent set_vars {
+  vars:
+      #Tracking key
+      &TRACKINGKEY:{key | "trackingkey[&i&]" string => "&key&";}&
+      #Shared files
+      "directory_server" string => "&SHARED_FILES_FOLDER&";
+      "directory_agent" string => "/var/rudder/resources";
```

Comment:
use a directory similar to the one used for templates, ex: /var/rudder/tmp/variables